punx/tests/test_issue133.py

```diff
@@ -48,7 +48,7 @@ def test_environment_restriction():
 
         # test that _some_ restriction exists
         assert len(item) > len("h5py")
-        req = item[len("h5py"):]
+        req = item[len("h5py"):].strip()
```

Comment:
What does this do?

Reply:
The `strip` allows for the [optional whitespace](https://github.com/prjemian/punx/commit/1e44450769a47d23124b4024534497832c82a2b0) before the pinned requirement: `h5py >=3`